src/components/policy/policy_external/src/policy_helper.cc

```diff
@@ -769,16 +786,23 @@ bool FillNotificationData::RpcParametersEmpty(RpcPermissions& rpc) {
          no_user_disallowed_parameters;
 }
 
-bool FillNotificationData::IsSectionEmpty(ParameterPermissions& permissions,
-                                          const std::string& section) {
+bool FillNotificationData::IsSectionEmpty(
+    const ParameterPermissions& permissions, const std::string& section) const {
   ParameterPermissions::const_iterator it_section = permissions.find(section);
   ParameterPermissions::const_iterator end = permissions.end();
   if (end != it_section) {
-    return permissions[section].empty();
+    return it_section->second.empty();
   }
   return true;
 }
 
+bool FillNotificationData::IsSomeParametersAllowed(
+    const ParameterPermissions& permissions) const {
+  return permissions.any_parameter_allowed ||
+         (kAllowedKey == current_key_ &&
+          !IsSectionEmpty(permissions, kAllowedKey));
```

Comment:
@AKalinich-Luxoft Really hard to read this return statement

Reply:
@AByzhynar splitted into two lines with the named consts
@conlain-k fixed in https://github.com/smartdevicelink/sdl_core/pull/2406/commits/80291097033c8a6cd44a95f2ff4ff4def57e5e8e